src/interface/obsidian/src/utils.ts

```diff
@@ -28,17 +28,42 @@ function fileExtensionToMimeType (extension: string): string {
     }
 }
 
-export async function updateContentIndex(vault: Vault, setting: KhojSetting, lastSyncedFiles: TFile[], regenerate: boolean = false): Promise<TFile[]> {
+function filenameToMimeType (filename: TFile): string {
+    switch (filename.extension) {
+        case 'pdf':
+            return 'application/pdf';
+        case 'png':
+            return 'image/png';
+        case 'jpg':
+        case 'jpeg':
+            return 'image/jpeg';
+        case 'md':
+        case 'markdown':
+            return 'text/markdown';
+        case 'org':
+            return 'text/org';
+        default:
+            return 'text/plain';
```

Comment:
hmm, is this a fair assumption? there would be a bunch of file types we can't process, and we wouldn't necessarily want them to slip through if they're interpreted as `text/plain`. like docx, xml, odt, ppt, etc.

Reply:
Yeah, this is the same function as the one we use in desktop `main.js`. I've updated the code to `console.warn` when we fallback to the default option for now